services/frontend/src/scenes/ProjectDetails/ProjectDetailsLayout.js

```diff
@@ -103,111 +103,92 @@ const TextBadge = ({className, text, isHighlighted = false}) => (
+                sequat. Officia aliqua ut reprehenderit ex occaecat ut aute dolor amet deserunt veniam. Reprehende
+                rit Lorem laboris est consequat. Enim ipsum ea do esse non esse incididunt id deserunt elit except
+                eur adipisicing ea irure. Elit voluptate cupidatat anim sit aute non excepteur Lorem nostrud occae
+                cat irure ut esse fugiat. Veniam proident esse aliqua do mollit laboris dolor. Adipisicing est nis
+                i id nisi nisi amet anim nostrud eiusmod ad fugiat qui.
+
+                Occaecat reprehenderit fugiat qui ullamco ad commodo Lorem velit nisi aliquip sit esse officia con
+                sequat. Officia aliqua ut reprehenderit ex occaecat ut aute dolor amet deserunt veniam. Reprehende
+                rit Lorem laboris est consequat. Enim ipsum ea do esse non esse incididunt id deserunt elit except
+                eur adipisicing ea irure. Elit voluptate cupidatat anim sit aute non excepteur Lorem nostrud occae
+                cat irure ut esse fugiat. Veniam proident esse aliqua do mollit laboris dolor. Adipisicing est nis
+                i id nisi nisi amet anim nostrud eiusmod ad fugiat qui.
+            </CardContent>
+        </Card>
     </>
 );
 
-const UsedSkills = ({sectionName}) => (
+const UsedSkills = ({sectionName, project}) => (
     <>
         <h2>{sectionName}</h2>
-        <Paper className="project-details-card">
-            Occaecat reprehenderit fugiat qui ullamco ad commodo Lorem velit nisi aliquip sit esse officia con
-            sequat. Officia aliqua ut reprehenderit ex occaecat ut aute dolor amet deserunt veniam. Reprehende
-            rit Lorem laboris est consequat. Enim ipsum ea do esse non esse incididunt id deserunt elit except
-            eur adipisicing ea irure. Elit voluptate cupidatat anim sit aute non excepteur Lorem nostrud occae
-            cat irure ut esse fugiat. Veniam proident esse aliqua do mollit laboris dolor. Adipisicing est nis
-            i id nisi nisi amet anim nostrud eiusmod ad fugiat qui.
-
-            Occaecat reprehenderit fugiat qui ullamco ad commodo Lorem velit nisi aliquip sit esse officia con
-            sequat. Officia aliqua ut reprehenderit ex occaecat ut aute dolor amet deserunt veniam. Reprehende
-            rit Lorem laboris est consequat. Enim ipsum ea do esse non esse incididunt id deserunt elit except
-            eur adipisicing ea irure. Elit voluptate cupidatat anim sit aute non excepteur Lorem nostrud occae
-            cat irure ut esse fugiat. Veniam proident esse aliqua do mollit laboris dolor. Adipisicing est nis
-            i id nisi nisi amet anim nostrud eiusmod ad fugiat qui.
-
-            <br /><br />
-        </Paper>
+        <Card className="project-details-card">
+            <CardContent className="project-used-skills-card">
+                <SkillBadges
+                    displayCount={project.skills.length}
```

Comment:
I think, instead of having to pass in the full length every time, that it'd be a better API to instead take the full length as the default, and only override that when `displayCount` is present.

As such, you should modify `SkillBadges` such that it could be used here as just:

```suggestion
                    <SkillBadges skills={project.skills} />
```